shared/tests/utility/math/docs/README.txt

```diff
@@ -0,0 +1,14 @@
+The files in this folder contain the same input data used in TestCoordinates.cpp along with some matlab scripts that perform the same calculations found in coordinates.h
+The output from matlab is used to validate the coordinates.h calculations in the NUBOTS codebase.
+Test input data can be found in both matlab and c++ formats. The matlab format data is passed into the functions starting with calc_****.m to produce the output data.
+Test output data provided is in a cpp format.
+
+NOTE: During initial testing when values at the extremes of the data types domain, DBL_MAX and DBL_MIN, were found to cause inf return values when passed to the functions
```

Comment:
did `DBL_MIN` give `inf` return values? wtf?

Reply:
I'm going to have to double check this.

Reply:
The extremely large and small numbers will often cause numerical instability. I'm not sure it makes sense to use them in lots of these tests as you will get nonsensical results. You should mainly make sure that values that are within our domain and range end up with sensible answers.

Reply:
This begs the question - how do we define or find out what our domain and range are for a given module?